docs/updates/2019-03.md

```diff
@@ -0,0 +1,47 @@
+# March 2019
+
+Welcome to the March 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Engine security enhancement
+
+Because of a security concern and to follow best-practices, the Qlik Associative Engine Docker image is changing its
+user and group from `nobody:nobody` to `1910:1910`. You can read more about using an unique UID/GID here:
+[https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user).
+
+If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
```

Comment:
Maybe make it bold as we did with https://core.qlik.com/updates/2019-01/#abac-moving-from-experimental-to-stable ?